third-party-podspecs/react-native-keyboard-aware-scroll-view.podspec.json

```diff
@@ -1,16 +1,19 @@
 {
   "name": "react-native-keyboard-aware-scroll-view",
-  "version": "0.8.8",
-  "homepage": "https://github.com/wordpress-mobile/react-native-keyboard-aware-scroll-view",
-  "summary": "React Native module to arrange scroll poisition according to keyboard on input fields.",
+  "version": "0.8.8-wp",
+  "homepage": "https://github.com/APSL/react-native-keyboard-aware-scroll-view#readme",
```

Comment:
Nitpick: If the source points to the WordPress-mobile repo, I'm wondering if we should do the same for the homepage 🤔 .

Reply:
I think the homepage should stay the same to indicate it's a fork and we would ideally want to merge the changes back to the original repo. The only reason we're modifying the source is to make WPiOS use the correct native source files from the fork if there were any changes made or if the original repo didn't specify a tag or a commit.

Reply:
Agreed, let's keep the original homepage 👍 .